RadioItem/RadioItem.module.less

```diff
@@ -5,6 +5,18 @@
 
 // Public
 .radioItem {
+	.icon {
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		box-sizing: border-box;
+	}
+
+	.slotBefore {
+		display: flex;
+		align-items: center;
+	}
```

Comment:
I couldn't find out why these 2 lines were added. Can you give more details?

Reply:
Well, by giving icon a width and height, when a `slotBefore` icon is used, the RadioItem icon and slotBefore icon are not aligned correctly. This was the fix I came up with